ci-operator/step-registry/quay-tests/test-quay-e2e/quay-tests-test-quay-e2e-commands.sh

```
@@ -1,119 +1,83 @@
 
-#Finally Copy the Junit Testing XML files and Screenshots to /tmp/artifacts
-trap copyArtifacts EXIT
-
 # Cypress Doc https://docs.cypress.io/guides/references/proxy-configuration
 if [ "${QUAY_PROXY}" = "true" ]; then
-    HTTPS_PROXY=$(cat $SHARED_DIR/proxy_public_url)
-    export HTTPS_PROXY
-    HTTP_PROXY=$(cat $SHARED_DIR/proxy_public_url)
-    export HTTP_PROXY
+    ( set +x
+        export HTTPS_PROXY="$(tr -d '\n' < "${SHARED_DIR}/proxy_public_url")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)